### DISCUSSION TITLE
"Confirm Merge" Button Doing Nothing

### DISCUSSION THREAD

**Discussion by Skagra42** [community/community#21587] [Pull Requests]
<p>I’m trying to complete the seventh step of the GitHub Actions: Hello World course and the “Confirm merge” button isn’t doing anything when I click it.</p>

**Comment by MCOfficer** [comment, marked as answer]
<p>Probably due to this: <a href="https://www.githubstatus.com/" rel="noopener nofollow ugc">https://www.githubstatus.com/</a></p>
<p>All we can do is wait.</p>
